test/common/config_parser_helper.erl

```diff
@@ -835,5 +836,57 @@ default_mod_config(mod_inbox) ->
       iqdisc => no_queue};
 default_mod_config(mod_private) ->
     #{iqdisc => one_queue, backend => rdbms};
+default_mod_config(mod_vcard) ->
+    #{iqdisc => parallel,
+      host => {prefix, <<"vjud.">>},
+      search => true,
+      backend => mnesia,
+      matches => 30,
+      ldap_pool_tag => default,
+      ldap_uids => [{<<"uid">>, <<"%u">>}],
+      ldap_vcard_map => [{<<"NICKNAME">>, <<"%u">>, []},
```

Comment:
I see that it's a small issue here - we can't reference MongooseIM code, because it needs to compile for big tests as well. I think it's ok as it is...

Reply:
Yes, this was exactly why I didn't reuse the macros from MIM code. This is not great, but it's a test helper, I figured it's best to keep it simple.